processing/src/main/java/org/apache/druid/segment/join/filter/JoinFilterAnalyzer.java

```diff
@@ -88,7 +88,8 @@ public static JoinFilterSplit splitFilter(
       HashJoinSegmentStorageAdapter hashJoinSegmentStorageAdapter,
       Set<String> baseColumnNames,
       @Nullable Filter originalFilter,
-      boolean enableFilterPushDown
+      boolean enableFilterPushDown,
+      boolean enableFilterRewrite
```

Comment:
Javadoc reference to this method on line 54 needs to be updated

Reply:
Fixed